core/src/replicator.rs

```diff
@@ -202,7 +202,7 @@ impl Replicator {
         node_info.tvu = "0.0.0.0:0".parse().unwrap();
         {
             let mut cluster_info_w = cluster_info.write().unwrap();
-            cluster_info_w.insert_info(node_info);
+            cluster_info_w.insert_self(node_info);
```

Comment:
@sakridge, added you to this just to make sure this info being added is indeed "self" info with an updated tvu?

Reply:
Yes, that's correct.